### ISSUE TITLE
Symlinks not working when using docker from Windows

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#1051]
The symlinks appear to be created with backward slashes instead of forward slashes. It might be related to this: https://github.com/docker/for-win/issues/109

Things to note:
- This worked fine in Vagrant (somehow)
- What are the exact settings of the Drini's machine wrt to git/symlinks/docker?
- It might be best to just stop using symlinks; they cause a lot of issues for Windows users.

**Comment by cdrini** [comment]
Here's some code to fix the broken symlinks in `/static/js` (run it from that directory)

```sh
for i in $(ls -l . | grep ^l | awk '{print $9}'); do
    target=`readlink $i`;
    target=${target//\\//}
    echo ln -s "$i" "$target"
done;
```

**Comment by hornc** [comment]
I'm not a fan of committing symlinks to git, even though I mainly use Linux. I have not dug into this, but can we fix this issue by removing the need for the symlinks altogether?

**Comment by cdrini** [comment]
Removing the need for symlinks would fix the issue. I am not entirely sure how I feel about committing symlinks, but I do know they've caused me a decent amount of trouble not just here as a Windows user.

**Comment by salman-bhai** [comment]
How do we approach to resolve this in that case then ?

**Comment by cdrini** [comment]
This appears to have been fixed by the new volume mounts in #1133 . Some issues might still exist if symlinks appear elsewhere in the codebase, but we can open new issues once those problems are found.
